### PR TITLE
TIaaS feedback post for our May workshop

### PR DESCRIPTION
**Pull request by mblue9**
Here's a feedback post if you want it (happy to make any edits).

### CODE REVIEW

File: _posts/2019-05-20-tiaas-feedback.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+site: freiburg
+tags: [training, tiaas]
+title: "Another successful TIaaS workshop"
```
Comment (bgruening) [inline _posts/2019-05-20-tiaas-feedback.md]: `Another successful Galaxy workshop in Melbourne` Maybe? Not sure the readers will know what TIaaS is ...

Comment (erasche) [inline _posts/2019-05-20-tiaas-feedback.md]:
```suggestion
title: "Another successful Galaxy workshop in Melbourne"
```

**Comment by mblue9** [comment]
Ok have made that change, thanks.

**Comment by bgruening** [comment]
Merci!!! :)

Those posts help us a lot!

**Pull request merged**